packages/zoe/src/contractSupport/durability.js

```diff
@@ -55,6 +57,16 @@ export const makePublicTopicProvider = () => {
       // @ts-expect-error cast
       return extant.get(durableSubscriber);
     }
+    mustMatch(
+      harden({ description, durableSubscriber, storageNode }),
+      harden(
+        harden({
```

Comment:
I don't think there's any value in nested `harden()` calls. Am I missing something?

Reply:
nope, a byproduct of refactoring. I'll fixup for merge